Neos.Cache/Classes/Backend/AbstractBackend.php

```diff
@@ -50,28 +50,25 @@ abstract class AbstractBackend implements BackendInterface
     protected $defaultLifetime = 3600;
 
     /**
-     * @var EnvironmentConfiguration
+     * @var ?EnvironmentConfiguration
      */
     protected $environmentConfiguration;
 
     /**
      * Constructs this backend
      *
-     * @param EnvironmentConfiguration $environmentConfiguration
-     * @param array $options Configuration options - depends on the actual backend
+     * @param ?EnvironmentConfiguration $environmentConfiguration @todo is this ever null and does that even make sense?
```

Comment:
fyi solved this in an followup pr https://github.com/neos/flow-development-collection/pull/3470